src/screens/FeedScreen/Header.js

```diff
@@ -102,15 +113,40 @@ const _renderAnonimity = ({
   height,
   headerStyle,
   showAnonymousOption = false,
-  onHeaderOptionClicked = () => {},
-  hideThreeDot,
   version = 1,
   anonUserInfo = {},
   isPostDetail,
   karmaScore = 0,
-  isBlurredPost = false
+  isBlurredPost = false,
+  isFollow = false,
+  onPressFollUnFoll = () => {},
+  onDeletePost = () => {},
+  isShowDelete = false,
+  isSelf = false,
+  hideThreeDot
 }) => {
   const navigation = useNavigation();
+  const refSheet = React.useRef();
+  const dataSheet = [
+    {
+      id: 1,
+      name: 'Share link',
+      icon: <ShareAndroidIcon />
+    }
+  ];
+
+  if (isShowDelete) {
+    dataSheet.push({
+      id: 2,
+      name: 'Delete post',
+      icon: <TrashRed />,
+      onPress: () => {
+        refSheet.current.close();
+        onDeletePost();
+      },
+      style: {color: COLORS.red}
+    });
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetMenu` component is being used here to display a menu with options. However, the data for this menu is being constructed within the component itself. This could lead to issues if you want to reuse this component elsewhere with different options. It would be better to pass the data as a prop to the component, making it more reusable and flexible.
```diff
-  const dataSheet = [
-    {
-      id: 1,
-      name: 'Share link',
-      icon: <ShareAndroidIcon />
-    }
-  ];
-
-  if (isShowDelete) {
-    dataSheet.push({
-      id: 2,
-      name: 'Delete post',
-      icon: <TrashRed />,
-      onPress: () => {
-        refSheet.current.close();
-        onDeletePost();
-      },
-      style: {color: COLORS.red}
-    });
-  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->